src/components/Table/Table.style.ts

```diff
@@ -19,6 +21,6 @@ export const tableRowHeadersStyle = () => (theme: Theme): SerializedStyles =>
     },
 
     'th:last-child': {
-      paddingRight: theme.spacing.md,
+      paddingRight: hasExpandableRows ? 'inherit' : theme.spacing.md,
```

Comment:
here also work the `hasExpandableRows && theme.spacing.md

Reply:
Tried it and TS throws an error.

`TS2769: No overload matches this call.`